Makefile

```diff
@@ -84,12 +93,17 @@ install: libutf8proc.a libutf8proc.$(SHLIB_EXT) libutf8proc.$(SHLIB_VERS_EXT)
 	mkdir -m 755 -p $(DESTDIR)$(libdir)
 	$(INSTALL) -m 644 libutf8proc.a $(DESTDIR)$(libdir)
 	$(INSTALL) -m 755 libutf8proc.$(SHLIB_VERS_EXT) $(DESTDIR)$(libdir)
-	ln -f -s $(libdir)/libutf8proc.$(SHLIB_VERS_EXT) $(DESTDIR)$(libdir)/libutf8proc.$(SHLIB_EXT)
+	cd $(DESTDIR)$(libdir) && ln -f -s libutf8proc.$(SHLIB_VERS_EXT) libutf8proc.$(SHLIB_EXT)
 ifneq ($(OS),Darwin)
-	ln -f -s $(libdir)/libutf8proc.$(SHLIB_VERS_EXT) $(DESTDIR)$(libdir)/libutf8proc.so.$(MAJOR)
-	ln -f -s $(libdir)/libutf8proc.$(SHLIB_VERS_EXT) $(DESTDIR)$(libdir)/libutf8proc.so.$(MAJOR).$(MINOR)
+	cd $(DESTDIR)$(libdir) && ln -f -s libutf8proc.$(SHLIB_VERS_EXT) libutf8proc.so.$(MAJOR)
 endif
 
+MANIFEST.new:
+	rm -rf INSTALL
```

Comment:
Probably call it `tmp` or similar rather than `INSTALL`.

(In GNU packages, `INSTALL` is a file with installation instructions.)